src/main/java/net/dries007/tfc/objects/blocks/devices/BlockFirePit.java

```diff
@@ -276,25 +289,11 @@ else if (tile.getCookingPotStage() == TEFirePit.CookingPotStage.FINISHED)
         return true;
     }
 
-    @Override
-    public void onEntityWalk(World worldIn, BlockPos pos, Entity entityIn)
-    {
-        IBlockState state = worldIn.getBlockState(pos);
-        if (state.getValue(LIT) && !entityIn.isImmuneToFire() && entityIn instanceof EntityLivingBase)
-        {
-            entityIn.attackEntityFrom(DamageSource.IN_FIRE, 1.0F);
-        }
-        super.onEntityWalk(worldIn, pos, entityIn);
-    }
-
     @Override
     public void onEntityCollision(World worldIn, BlockPos pos, IBlockState state, Entity entityIn)
     {
-        if (state.getValue(LIT) && !entityIn.isImmuneToFire() && entityIn instanceof EntityLivingBase)
-        {
-            entityIn.attackEntityFrom(DamageSource.IN_FIRE, 1.0F);
-        }
-        super.onEntityWalk(worldIn, pos, entityIn);
+        //todo: handle fuel and item inputs from thrown entities
+        super.onEntityCollision(worldIn, pos, state, entityIn);
```

Comment:
The point of `isBurning`: It burns every entity it touches. It's also hardcoded to entity movement as it asks the `World` if it should be taking fire damage. So this `//todo` is rather meaningless, and so there's no reason to have this override here.

Reply:
If the isBurning changes are reverted, this todo would be valid then? This is also a fix of previous copypasta.